airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/LowCreditBalanceHint/LowCreditBalanceHint.module.scss

```diff
@@ -0,0 +1,28 @@
+@use "../../../../../../../scss/colors";
+
+.container {
+  width: 100%;
+
+  &.zeroBalance {
+    background-color: colors.$red-50;
+
+    & > svg {
+      color: colors.$red-300;
+    }
+  }
+
+  &.lowBalance {
+    background-color: colors.$yellow-50;
```

Comment:
I'm not sure we should really diverge from our regular background color that `InfoBox` has here. I think it would be better if we stay aligned with the color and not trying to overwrite it. I'd also recommend not overwriting the icon color for the same reason (not to end up with too many unaligned styles around the app).

Reply:
I've followed the colors that we have in [Figma](https://www.figma.com/file/gMNRYVJav25pdsoxAyW7Qp/00_01_WEB-APP-LIBRARIES?node-id=1833%3A3458&t=HDnTXz1PvdIMoXap-0) for this component (both bg and icon). But happy to consolidate styling.
Should I follow the styles on the figma or stick with the existing ones in code?

Reply:
@letiescanciano good catch, IMO that looks like a design inconsistency that we should address with Nico. But I think the component you linked is rather a toast message than an `InfoBox`, right? For now, can we keep the existing colors we have in code (i.e. `yellow-100`)?

Reply:
There's two actually Toast and Message. It was the closest to an `Infobox` I could find but agree, let's keep the ones in code for consistency 👍🏼